### PR TITLE
more fixes to the rpc

### PR DESCRIPTION
**Pull request by alloncm**
a few more things I haven't noticed :(

### CODE REVIEW

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs [hunk 1]
```diff
@@ -12,10 +12,12 @@ public class NetqRabbitRpcConnection : INetqRabbitRpcConnection
     {
         public IBus Bus { get; }
         public RpcMessageType RpcMessageType { get; }
+        private readonly bool _routingKeyProviderExist = false;
+        private readonly ITypeNameSerializer _typeNameSerializer = new DefaultTypeNameSerializer();
```
Comment (ybs007) [inline Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs]: Initialize them in the construcor

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs [hunk 2]
```diff
@@ -28,11 +28,11 @@ public NetqRabbitRpcServer(INetqRabbitRpcConnection netqRpcRabbitConnection, ILo
             VerifyServerHasntStartedYet();
             if (_netqRabbitRpcConnection.RpcMessageType == RpcMessageType.Abstract)
             {
-                _respondHandler = _netqRabbitRpcConnection.Bus.Respond<TRequest, TRespond>(callback);
```
Comment (ybs007) [inline Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs]: Why you moved the TRequest, Trespond here but kept it in other places , be consistent about it, i think that explicit is better

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs [hunk 3]
```diff
@@ -54,11 +54,11 @@ public NetqRabbitRpcServer(INetqRabbitRpcConnection netqRpcRabbitConnection, ILo
             VerifyServerHasntStartedYet();
             if (_netqRabbitRpcConnection.RpcMessageType == RpcMessageType.Abstract)
             {
-                _respondHandler = _netqRabbitRpcConnection.Bus.RespondAsync<TRequest, TRespond>(callback);
```
Comment (ybs007) [inline Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs]: Why you moved the TRequest, Trespond here but kept it in other places , be consistent about it, i think that explicit is better

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs (hunk 1 above)
Comment (alloncm) [reply Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs]: f

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs (hunk 2 above)
Comment (alloncm) [reply Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs]: f

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs (hunk 3 above)
Comment (alloncm) [reply Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcServer.cs]: f

File: Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs (hunk 1 above)
Comment (ybs007) [reply Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs]: You didn't fix it

Comment (alloncm) [reply Blizzard.IO.RabbitMQ/Rpc/NetqRabbitRpcConnection.cs]: f for real

**Pull request merged**